component/Input/input.css

```diff
@@ -0,0 +1,117 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+.wrapper{
+    display: flex;
+    justify-content: center;
+    margin-bottom: 2rem;
+
+}
+form{
```

Comment:
Dont style the element directly. Add a class to that element and style it

Reply:
ok